Sources/FireworkVideoUI/AppLanguage/Extensions/UIKit/UIView+AppLanguage.swift

```diff
@@ -8,6 +8,10 @@ import UIKit
 import FireworkVideo
 import AVFoundation
 
+private let gNamesOfImagesWithDirection: [String] = [
+    "c3RyZWFtLWdhdGUtYmFjaw==".decodeBase64String(),
+]
```

Comment:
This base64 decoded string seems to be referencing and internal image of the FireworkVideoSDK. This is extremely fragile. What is the intent on this?

Reply:
I didn't find the other way instead of hardcoding FireworkVideoSDK image names here. The image is used in SwiftUI. We don't have another way to check whether the image should be flipped. For the image used in UIImageView, we could check the `flipsForRightToLeftLayoutDirection` property. But we can't find the related property in SwiftUI.